collect_app/src/test/java/org/odk/collect/android/utilities/ValidatorTest.java

```diff
@@ -55,4 +55,58 @@ public void emailValidationTestCase() {
         assertFalse(Validator.isEmailAddressValid("email@"));
         assertFalse(Validator.isEmailAddressValid("email.domain.com"));
     }
+
+    @Test
+    public void urlValidationTest() {
```

Comment:
If the test is from https://en.wikipedia.org/wiki/Template:URL/testcases maybe you should add a link here you know just for copyrights.

Reply:
Sure!